mxcube3/ui/components/Tasks/Characterisation.js

```diff
@@ -188,7 +190,7 @@ Characterisation = connect(state => {
   const subdir = selector(state, 'subdir');
   const prefix = selector(state, 'prefix');
   const runNumber = selector(state, 'run_number');
-  const fileSuffix = state.taskForm.fileSuffix === 'h5' ? '_master.h5' : '_????.cbf';
+  const fileSuffix = state.taskForm.fileSuffix === 'h5' ? '_master.h5' : 'cbf';
```

Comment:
Hey, actually there is not supposed to be a dot here, its just the extension without (.)